test/bats/e2e-provider.bats

```diff
@@ -58,8 +68,8 @@ export LABEL_VALUE=${LABEL_VALUE:-"test"}
   assert_success
 }
 
-@test "deploy e2e-provider secretproviderclass crd" {
-  envsubst < $BATS_TESTS_DIR/e2e_provider_v1alpha1_secretproviderclass.yaml | kubectl apply -f -
+@test "deploy e2e-provider v1 secretproviderclass crd" {
+  envsubst < $BATS_TESTS_DIR/e2e_provider_v1_secretproviderclass.yaml | kubectl apply -f -
```

Comment:
I think we will want at least 1 e2e-provider test case where the CRD is the v1alpha1 version to demonstrate that the new release still of the driver continues to work with the v1alpha1 CRD. Maybe you did this but its hard to tell from the diff

Reply:
The `v1alpha1` to `v1` upgrade tests are covered as part of upgrade and inplace upgrade tests. Added another set of tests just to validate with `v1alpha1` api version.